pset-functions.js

```diff
@@ -91,6 +107,23 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+function wordCount(phrase) {
+   //Defining the Word array
+   const cleanedPhrase = phrase.replace(/[,]/g, '')
+   //Defining blank array for later use
+   const wordArray = {}
+
+   //Looping through array and assigning number of instances to each element
+   cleanedPhrase.split(" ").forEach(function(currentValue, i, array) {
+      wordArray[currentValue] = wordArray[currentValue]? ++ wordArray[currentValue]: 1
+   })
+   //Printing final array
+   return wordArray
+}
+
+const finalPhrase = wordCount("Baby shark, doo doo doo doo doo doo")
+console.log(finalPhrase)
+
```

Comment:
@cswebdesign Great work here with this solution. Nice job using the conditional ternary operator to increment the object value by 1 if it exists and creating the key and setting the value to 1 if it doesn't.

Take a look at this solution [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L103)